cfg/rationalize.go

```diff
@@ -86,6 +86,12 @@ func resolveStreamingWriteConfig(w *WriteConfig) {
 	}
 }
 
+func resolveCloudMetricsUploadIntervalSecs(m *MetricsConfig) {
+	if m.CloudMetricsExportIntervalSecs == 0 {
+		m.CloudMetricsExportIntervalSecs = int64(m.StackdriverExportInterval.Seconds())
```

Comment:
what happens if stackDriverExportInterval is not set

Reply:
It'll be 0 by default.